diff --git a/include/mach-o/dyld.h b/include/mach-o/dyld.h
index a2e641c..f0036f7 100644
--- a/include/mach-o/dyld.h
+++ b/include/mach-o/dyld.h
@@ -118,6 +118,7 @@ extern void _tlv_bootstrap(void)                                             __O
  * cannot stat() the file to see if it "exists".  This function is like a stat() call that checks if a
  * path is to a dylib that was removed from disk and is incorporated into the active dyld cache.
  */
+#define __API_AVAILABLE_PLATFORM_bridgeos(x) bridgeos,introduced=x
 extern bool _dyld_shared_cache_contains_path(const char* path)               __API_AVAILABLE(macos(11.0), ios(14.0), watchos(7.0), tvos(14.0), bridgeos(5.0)) DYLD_DRIVERKIT_UNAVAILABLE;
 
 
@@ -159,6 +160,7 @@ typedef enum {
 
 typedef struct __NSObjectFileImage* NSObjectFileImage;
 
+#define __API_UNAVAILABLE_PLATFORM_bridgeos bridgeos,unavailable
 
 
 /* NSObjectFileImage can only be used with MH_BUNDLE files */
diff --git a/include/mach-o/dyld_priv.h b/include/mach-o/dyld_priv.h
index c2ba57c..1e70889 100644
--- a/include/mach-o/dyld_priv.h
+++ b/include/mach-o/dyld_priv.h
@@ -126,6 +126,8 @@ typedef struct {
     uint32_t        version;
 } dyld_build_version_t;
 
+#define __API_AVAILABLE_PLATFORM_bridgeos(x) bridgeos,introduced=x
+
 // Returns the active platform of the process
 extern dyld_platform_t dyld_get_active_platform(void) __API_AVAILABLE(macos(10.14), ios(12.0), watchos(5.0), tvos(12.0), bridgeos(3.0));
 
