diff --git c/build/macos.inc i/build/macos.inc
index dde93216..de912be8 100644
--- c/build/macos.inc
+++ i/build/macos.inc
@@ -38,6 +38,8 @@ ifndef arch
  else
    ifeq ($(shell /usr/sbin/sysctl -n hw.optional.x86_64 2>/dev/null),1)
      export arch:=intel64
+   else ifeq ($(shell /usr/sbin/sysctl -n hw.optional.arm64 2>/dev/null),1)
+     export arch:=arm64
    else
      export arch:=ia32
    endif
diff --git c/include/tbb/tbb_machine.h i/include/tbb/tbb_machine.h
index 9752be58..63a0e0b8 100644
--- c/include/tbb/tbb_machine.h
+++ i/include/tbb/tbb_machine.h
@@ -254,6 +254,8 @@ template<> struct atomic_selector<8> {
         #include "machine/linux_intel64.h"
     #elif __POWERPC__
         #include "machine/mac_ppc.h"
+    #elif __aarch64__
+        #include "machine/gcc_arm.h"
     #endif
     #include "machine/macos_common.h"
 
